packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,12 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [6.0.0]
-### Uncategorized
+### Changed
 - Use enum for apporoval type ([#1217](https://github.com/MetaMask/core/pull/1217))
```

Comment:
I don't think this one is needed as externally speaking, it's just part of the approval requests which is already mentioned below.